openapi_python_client/schema/openapi_schema_pydantic/external_documentation.py

```diff
@@ -12,11 +12,12 @@ class ExternalDocumentation(BaseModel):
     [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.
     """
 
-    url: AnyUrl
+    url: AnyUrl = ...  # type: ignore [assignment]
```

Comment:
Mypy do not like the ellipsis notation, flak8 is ok with it. I'm not too familiar with it, but after a quick looks on it, my current understanding is that flak8 is right, mypy is wrong, That why I've told mypy to ignore them

Reply:
I think that with Pydantic using the ellipsis is the same as not assigning anything, so should we just leave out the assignment?

Obviously the more differences there are with the vendored lib the harder it is to reconcile differences. Maybe this should be using a fork instead of a vendored version so that's easier?

Reply:
Yes, my understanding is that ellipsis notation is used to represent `undefined[T]`.  Agreed on removing it, resolved on https://github.com/triaxtec/openapi-python-client/pull/426/commits/b0dd28aacefdd4964bad517549d1182caa5c1394

Having a fork imo is a good idea, it start to have a reasonable amount of diff, even if they are small one